src/Features/Core/Portable/Completion/CompletionOptionsMetadata.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Completion
+{
+    // TODO: Move to EditorFeatures https://github.com/dotnet/roslyn/issues/59184
+    internal static class CompletionOptionsMetadata
```

Comment:
i might even call this CompletionOptionsStorage.  It's not really 'metadata' per se... it's more telling the sytem here's how we load/persist these options from varying locatinos, right?

Reply:
Yeah, storage might be a good name. I'd keep Metadata for now to keep consistency. We can do rename for all of these later on.

Reply:
Actually, I can just rename while moving to the new pattern.